AWSAuthSDK/AWSAuthSDK.xcodeproj/project.pbxproj

```diff
@@ -1475,26 +1499,40 @@
 				17CD4B22218244EA00953483 /* AppDelegate.swift */,
 				17CD4B24218244EA00953483 /* ViewController.swift */,
 				17CD4B26218244EA00953483 /* Main.storyboard */,
-				17CD4B29218244EC00953483 /* Assets.xcassets */,
 				17CD4B2B218244EC00953483 /* LaunchScreen.storyboard */,
+				17CD4B29218244EC00953483 /* Assets.xcassets */,
 				17CD4B2E218244EC00953483 /* Info.plist */,
 			);
 			path = AWSAuthSDKTestApp;
 			sourceTree = "<group>";
 		};
+		44F8C2D9236CB1D000D9C277 /* Fonts */ = {
+			isa = PBXGroup;
+			children = (
+				44F8C2DA236CB1D000D9C277 /* MavenPro-Black.ttf */,
+				44F8C2DB236CB1D000D9C277 /* MavenPro-Regular.ttf */,
+				44F8C2DC236CB1D000D9C277 /* MavenPro-Medium.ttf */,
+				44F8C2DD236CB1D000D9C277 /* MavenPro-Bold.ttf */,
+			);
+			name = Fonts;
+			path = "../../../../Users/ruslan/Desktop/aws-sdk-ios/AWSAuthSDK/Fonts";
```

Comment:
@ruslankuksa  this should be a relative path so it can work on all of our dev environments.